utils/format_doc.mli

```diff
@@ -167,6 +167,8 @@ val compat2: ('p1 -> 'p2 -> 'a printer) -> ('p1 -> 'p2 -> 'a format_printer)
 (** If necessary, embbed a {!Format} printer inside a formatting instruction
     stream. This breaks every guarantees provided by {!Format_doc}. *)
 val deprecated_printer: (Format.formatter -> unit) -> formatter -> unit
+val deprecated_printer1: 'a format_printer -> 'a printer
+val deprecated_printer2: ('p1 -> 'a format_printer) -> ('p1 -> 'a printer)
```

Comment:
I think it might be better to name those new functions `deprecated` and `deprecated1` since they are the exact counterpart of the `compat` and `compat1` functions.

Reply:
that's a good idea. Done